readme.md

```diff
@@ -83,6 +83,10 @@ repositories is also required.
 
 ## Limits
 
```

Comment:
Maybe some general information on why the limits are important. And maybe something about the "limits" when not working in Salesforce, like efficient code being important.